libenkf/src/gen_data.c

```diff
@@ -271,7 +273,9 @@ static bool gen_data_fload_active__(gen_data_type * gen_data, const char * filen
             util_abort("%s: error when loading active mask from:%s - file not long enough.\n",__func__ , active_file );
         }
         fclose( stream );
-      }
+        res_log_add_fmt_message( LOG_INFO , NULL , "GEN_DATA(%s): active information loaded from:%s.", gen_data_get_key( gen_data ), active_file);
+      } else
```

Comment:
If you feel in the mood, some early exists would be appreciated. If not, let it be ;)

Reply:
Hmmm - was not in the mood right now :-(